sdk/identity/identity/src/credentials/environmentCredential.ts

```diff
@@ -149,7 +145,7 @@ export class EnvironmentCredential implements TokenCredential {
 
     // If by this point we don't have a credential, throw an exception so that
     // the user knows the credential was not configured appropriately
-    span.setStatus({ code: StatusCode.ERROR });
```

Comment:
@daviwil: I noticed this while I was doing an openTelemetry upgrade. Ignoring that StatusCode has changed, is it appropriate for me to include this error message here? I assume that if it's worth throwing an exception for then it's worth adding in some context to the span as well.